File: kattis/judgingmoose.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int l, r;
    cin >> l >> r;
    if (l == 0 and r == 0) {
        cout << "Not a moose" << endl;
    }
    else if (l == r) {
        cout << "Even " << l * 2 << endl;
    }
    else {
        int highest = max(l, r);
        cout << "Odd " << highest * 2 << endl;
    }
}

File: kattis/judgingmoose.java
import java.util.*;

class JudgingMoose {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int l, r;
		l = scan.nextInt();
		r = scan.nextInt();
		if (l == 0 && r == 0) {
			System.out.print("Not a moose");
		}
		else {
			if (l == r) {
				System.out.print("Even ");
			} else {
				System.out.print("Odd ");
			}
			System.out.println(Math.max(l, r) * 2);
		}
	}
}